src/bust.py

```diff
@@ -197,7 +197,7 @@ async def play_song(self, index: int) -> None:
         submit_message, attachment, local_filepath = self.bust_content[index]
 
         # Associate a random emoji with this song
-        random_emoji = random.choice(config.emoji_list).encode("Latin1").decode()
+        random_emoji = random.choice(config.emoji_list)
```

Comment:
Doing this encoding and decoding dance is no longer necessary after the library [updated to encode unicode properly](https://github.com/sevenc-nanashi/discord-emoji/commit/7699082118ba22d4e007d1f48be1f47c48e32de4).